modules/tinymce/src/core/main/ts/fmt/ApplyFormat.ts

```diff
@@ -38,12 +40,34 @@ const isElementNode = function (node: Node) {
   return node && node.nodeType === 1 && !Bookmarks.isBookmarkNode(node) && !isCaretNode(node) && !NodeType.isBogus(node);
 };
 
+const canFormatBR = (editor: Editor, format: ApplyFormat, node: Node, parentName: string) => {
+  // TINY-6483: Can format 'br' if it is contained in a valid empty block and an inline format is being applied
+  if (Settings.canFormatEmptyLines(editor) && FormatUtils.isInlineFormat(format)) {
+    // A curated list using the textBlockElements map and parts of the blockElements map from the schema
+    const validBRParentElements: Record<string, {}> = {
```

Comment:
Should this be in the schema itself, rather than here?

Reply:
I did think about that but it doesn't really have any use anywhere else at the moment. If it needs to be used in another file then I think it might be worth doing.